packages/wonder-blocks-core/components/unique-id-provider.md

```diff
@@ -0,0 +1,101 @@
+The `UniqueIDProvider` component is how Wonder Blocks components obtain unique identifiers. This component ensures that server-side rendering and initial client rendering match while allowing the provision of unique identifiers for the client. It does this by not guaranteeing unique identifiers for the initial render, but instead allowing for a placeholder to be rendered in place of the real content, or for non-unique values to be provided as identifiers.
```

Comment:
My understanding is that `UniqueIDProvider` only provides a single unique identifier.  It might be worth calling that out here.